spec/datetimeformat.html

```diff
@@ -17,8 +17,7 @@ <h1>Intl.DateTimeFormat ( [ _locales_ [ , _options_ ] ] )</h1>
 
       <emu-alg>
         1. If NewTarget is *undefined*, let _newTarget_ be the active function object, else let _newTarget_ be NewTarget.
-        1. Let _dateTimeFormat_ be ? OrdinaryCreateFromConstructor(_newTarget_, *"%DateTimeFormat.prototype%"*, &laquo; [[InitializedDateTimeFormat]], [[Locale]], [[Calendar]], [[NumberingSystem]], [[TimeZone]], [[Weekday]], [[Era]], [[Year]], [[Month]], [[Day]], [[DayPeriod]], [[Hour]], [[Minute]], [[Second]], [[FractionalSecondDigits]], [[TimeZoneName]], [[HourCycle]], [[Pattern]], [[BoundFormat]] &raquo;).
-        1. Perform ? InitializeDateTimeFormat(_dateTimeFormat_, _locales_, _options_).
+        1. Let _dateTimeFormat_ be ? CreateDateTimeFormat(_newTarget_, _locales_, _options_, ~any~, ~date~).
```

Comment:
Why is `defaults` set to `~date~` and not `~all~` here?

Reply:
That's the preexisting behaviour of `Intl.DateTimeFormat`, cf. [InitializeDateTimeFormat, step 2](https://tc39.es/ecma402/#sec-initializedatetimeformat). 

```js
js> new Intl.DateTimeFormat("en", {timeZone: "UTC"}).format(new Date(0))
"1/1/1970"
```

Reply:
Got it, thanks.